s3/pkg/datastore/yig/tests/utils_test.go

```diff
@@ -0,0 +1,56 @@
+package tests
```

Comment:
Hi @hopkings2008  Please add a license announcement at the beginning. At the same time, add some notes to explain it used the snowflake. Thanks

Reply:
ok, very good suggestion @sfzeng , i will amend it according to your suggestion.